test_binaries/extract-module-sig.pl

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env perl
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
This license is different from what op-test uses. Is this okay?
Is it possible to download this file at runtime?

Reply:
Yes agree with sachin , please use this link https://raw.githubusercontent.com/torvalds/linux/master/scripts/extract-module-sig.pl , may have `wget` in test itself as we don't have any API in op-test to perform wget

Reply:
Ok, will change.